pyoptsparse/pyNSGA2/pyNSGA2.py

```diff
@@ -188,6 +188,12 @@ def objconfunc(nreal, nobj, ncon, x, f, g):
             for i in range(n):
                 xstar[i] = nsga2.doubleArray_getitem(x, i)
 
+            if len_ff > 1:
+                for i in range(len_ff):
+                    ff[i] = nsga2.doubleArray_getitem(f, i)
```

Comment:
Can you add some comments to explain what's happening here? I don't understand why `ff` has to be overwritten.

Reply:
If I am not mistaken, `ff` is just initialized (as a 0-length array) in `_assembleObjective`. Before this fix, `fStar` was never updated with the value of the optimal objective `f`, returned from the actual optimizer called by `nsga2.nsga2()`. 
Moreover, f is a swig object  so `nsga2.doubleArray_getitem` is necessary to convert it to a numpy array.